src/content/blog/es/canvas-cli-la-guia-completa.md

```
@@ -0,0 +1,853 @@
+courses=$(canvas courses list -o json | jq -r '.[].id')
+
+echo "$courses" | xargs -P 4 -I {} sh -c '
+  canvas enrollments list --course-id {} -o csv > inscripciones_{}.csv
+'
+```
+
+### 4. Entendiendo la Adaptación del Rate Limit
+
+Canvas CLI se adapta automáticamente a los límites de la API:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)